pkg/http-server/file-scan.go

```diff
@@ -37,6 +37,7 @@ func (g *APIHandler) scanFile(w http.ResponseWriter, r *http.Request) {
 	var (
 		iacType    = params["iac"]
 		iacVersion = params["iacVersion"]
+		severity   = params["severity"]
```

Comment:
`severity` would not be received in the url params

Reply:
then how?

Reply:
@kanchwala-yusuf changes made!